src/index.js

```diff
@@ -11,7 +11,7 @@ const setNanoid = url => {
         return url;
     }
 
-    let [uri, query] = fullUrl.split('?');
+    let [uri, query] = url.split('?');
```

Comment:
`fullUrl` variable is determined a little higher according to the conditions 
https://github.com/posthtml/posthtml-cache/blob/e5b5c2b331d543cea34e6e89127cb86f4aef538d/src/index.js#L8